flow-typed/npm/hash_v2.x.x.js

```diff
@@ -0,0 +1,11 @@
+// @flow
```

Comment:
Is there a way to put these declarations inside the actual package without creating a wrapper around the native module?

Reply:
Move this file to `packages/utils/hash/index.js.flow` and remove the `declare module` wrapper